scripts/sign_jwt.sh

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/bash
+
+# Inspired by implementation by Will Haley at:
+#   http://willhaley.com/blog/generate-jwt-with-bash/
+
+set -o pipefail
+
+apt-get -o DPkg::Lock::Timeout=60 update > /dev/null
+apt-get -o DPkg::Lock::Timeout=60 install -y jq xxd > /dev/null
+
+# Shared content to use as template
+header_template='{
+    "typ": "JWT",
+    "kid": "0001"
+}'
+
+build_header() {
+        jq -c \
+                --arg iat_str "$(date +%s)" \
+                --arg alg "${1}" \
+                '
+        ($iat_str | tonumber) as $iat
+        | .alg = $alg
+        | .iat = $iat
+        | .exp = ($iat + 86400)
+        ' <<<"$header_template" | tr -d '\n'
```

Comment:
If you use `--argjson iat_str "$(date +%s)"` you don't have to convert `$iat_str` to a number in the `jq` expression, or you could just use the built-in `now` in `jq` directly:

```suggestion
        jq -c \
                --arg alg "${1}" \
                '
        (now|round) as $iat
        | .alg = $alg
        | .iat = $iat
        | .exp = ($iat + 86400)
        ' <<<"$header_template" | tr -d '\n'
```